### DISCUSSION TITLE
Why there is no documetation on blocks packages?

### DISCUSSION THREAD

**Discussion by shadigaafar** [WordPress/gutenberg#35491] [Q&A]
Where is the documentation on @wordpress/components and other packages?.... How are we supposed to know the components inside of this package? I mean, it's illogical to go through all of the files of this package to discover the components. And look for the component that you want to use. I've always said that the Wordpress documentation is the worst. And the blocks documentation is the worst of the worst.

**Comment by tomalec** [comment]
You can find a storybook of all components at
https://wordpress.github.io/gutenberg/

I do agree the documentation is far from perfect and even harder to find.
Personally, I hope one day we could make it better, as I struggle with this as well.

**Comment by talldan** [comment]
There's some documentation here for each package:
https://developer.wordpress.org/block-editor/reference-guides/packages/

And also README files in each package folder (and worth looking for README within the codebase).

**Comment by ryanwelcher** [comment]
@shadigaafar there is an effort underway to update the documentation for the components package in #33263. There's a lot to do and we'd love the help if you're interested. There is an [example section](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/CONTRIBUTING.md#readme-example) in the CONTRIBUTON.md file that provides a starting point and removes much of the guesswork around formatting etc.
